cypress/support/fragments/settings/notes/noteTypes.js

```diff
@@ -52,32 +57,38 @@ export default {
     cy.expect(saveNoteTypeButton.has({ disabled: false }));
   },
 
-  saveNoteType: (noteType) => {
+  saveNoteType(noteType) {
     cy.do(saveNoteTypeButton.click());
     // need to wait for note type to appear after creation
     cy.wait(2000);
-    cy.expect(MultiColumnListRow({ content: including(noteType) }).exists());
+    this.checkNoteTypeIsDisplayed(noteType);
+  },
+
+  deleteNoteType(noteType) {
```

Comment:
Could you please add more checks here (according to test case):
- that Cancel and Delete buttons shown in delete modal,
- that deleted note type is not shown in the list?
_In this method, or as separate methods to use in spec file_

Reply:
fixed